<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrialsEditorTool</class>
 <widget class="QMainWindow" name="TrialsEditorTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrialsEditorTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="allWidgetsLayout">
      <item>
       <layout class="QHBoxLayout" name="gameSelectionLayout">
        <item>
         <widget class="QRadioButton" name="risingRadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Rising</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">gameSelectionButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fusionRadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Fusion</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">gameSelectionButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="trackFolderSelectionLayout">
        <item>
         <widget class="QLabel" name="dirLabel">
          <property name="text">
           <string>Track folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="selectDirLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="favoritesButton">
          <property name="text">
           <string>Downloaded</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="trackListsLayout">
        <item>
         <layout class="QVBoxLayout" name="availableTracksLayout">
          <item>
           <widget class="QLabel" name="availableLabel">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>Available tracks</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="availableTracksList">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="listControlButtonsLayout">
          <item>
           <widget class="QPushButton" name="addTrackButton">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeTrackButton">
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportTrackButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="exportTracksLayout">
          <item>
           <widget class="QLabel" name="exportLabel">
            <property name="text">
             <string>Export</string>
            </property>
            <property name="indent">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="exportTracksList">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="gameSelectionButtonGroup"/>
 </buttongroups>
</ui>
